# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-mochitest/test_has_rbab.html
# Commit: 7e2a2b1b416f
# Full Hash: 7e2a2b1b416f1a64c6548aab29e92acc9554ae7d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix all webgl regression tests according to CI. r=handyman
#   
#   (This is a combination of 31 commits)
#   
#   * Fix Linux compilation.
# ==============================================================================

diff -r ccfa767dba64 -r 7e2a2b1b416f dom/canvas/test/webgl-mochitest/test_has_rbab.html
--- a/dom/canvas/test/webgl-mochitest/test_has_rbab.html	Wed Jan 08 22:19:16 2020 +0000
+++ b/dom/canvas/test/webgl-mochitest/test_has_rbab.html	Wed Jan 08 22:19:23 2020 +0000
@@ -16,6 +16,8 @@
   ok(e, 'Should have MOZ_debug.');
 
   const does_index_validation = e.getParameter(e.DOES_INDEX_VALIDATION);
+  const err = g.getError();
+  ok(!err, 'Error should be 0, was 0x' + err.toString(16));
   ok(!does_index_validation,
      "Should have RBAB. Mark this test as failing on platforms that can't support it.");
 